commands/operator-sdk/cmd/test/local.go

```diff
@@ -141,6 +149,11 @@ func testLocalFunc(cmd *cobra.Command, args []string) {
 			}
 		}()
 	}
+	if tlConfig.image != "" {
+		if err := replaceImage(tlConfig.namespacedManPath, tlConfig.image); err != nil {
+			log.Fatalf("replaceImage function failed: %v", err)
```

Comment:
Probably rephrase this to be better understood by the user:
```Go
log.Fatalf("failed to overwrite operator image in the namespaced manifest: %v", err)
```

Reply:
Yup, forgot to reword that after debugging.